src/lib/wallet.ts

```diff
@@ -401,7 +401,7 @@ export const makeWalletUtils = async (rpcUtils: RpcUtils, keplr: Keplr) => {
 };
 
 const usp = new URLSearchParams(window.location.search);
-const agoricNet = usp.get('agoricNet') || 'devnet';
+const agoricNet = usp.get('agoricNet') || 'main';
```

Comment:
I don't understand the diagnosis that leads to this fix, but it can't hurt much.